### ISSUE TITLE
Add GTFS feeds for Addis Ababa

### ISSUE THREAD

**Issue by FloEdelmann** [osm-ToniE/gtfs-feeds#31]
* GitLab repo with GTFS feeds: https://gitlab.com/digitaltransport/data/africa/addis-ababa/-/tree/master/Addis%20Ababa%20GTFS_2023?ref_type=heads
* Feed in Transitland: https://www.transit.land/feeds/f-addis~ababa
* License: ODbL
* Press releases:
  * https://digitaltransport4africa.org/dt4a-repository-received-gtfs-datasets-from-addis-ababa/
  * https://www.trufi-association.org/addismaptransit-submits-transport-routes-to-dt4a/

I'm not sure if they are useful or already used at https://ptna.openstreetmap.de/results/ET/index.php, but I wanted to share it with you anyway.

**Comment by osm-ToniE** [comment]
Thanks,

I had a closer look into the data and, from PTNA's automation process point of view, the data misses important information.

- `curl -sI "https://gitlab.com/digitaltransport/data/africa/addis-ababa/-/raw/master/Addis%20Ababa%20GTFS_2023/et-addisababa.zip"` does not show the "last modified" header for the data
- when downloading the *.zip, the file date stamps show "01.01.1980" which is also not helpful
  -  I want to avoid downloading the *.zip just to see the creation/modification date 

So I can't figure out when the data has been created / last modified. This is essential for PTNA.

**Comment by FloEdelmann** [comment]
> `curl …` does not show the "last modified" header for the data

Indeed, that seems to be a missing GitLab feature. But I see that they send an `etag` header, which could be used to check for changes since the last fetch. But I understand if you don't want to implement that extra check only for this feed. So feel free to just close this issue.

**Comment by osm-ToniE** [comment]
Yeah, the `etag` could be an option, but the `release-date` (defined by the feed owner) is essential for PTNA.

So, PTNA could read the release-date either
- from the "last-modified" HTTP header
  - as suggest by "best practice for GTFS"
- or the time stamps (GNU tool `stat -c '%y' filename`) of the files in the *.zip
  - having always 1.1.1980 here is "bad coding" for me
- or a time stamp in the feed_info.txt column '`feed_version`'
  - which is "1.0" at the moment

**Comment by FloEdelmann** [comment]
I created an issue: https://gitlab.com/digitaltransport/data/africa/addis-ababa/-/issues/1
Let's see if they can change anything to make this usable in PTNA.
